### PR TITLE
Add emsys grid logo to first page

### PR DESCRIPTION
**Pull request by So-Fras**
**Please check if the PR fulfills these requirements**

- [x] The commit message follows our guidelines

**What kind of change does this PR introduce?**
Add emsys grid logo, link and description to first page sections "They use PowSyBl" and "Partners and vendors".

A GitHub link is missing and shall be added when it is made available (maybe in another PR).

**Does this PR introduce a breaking change or deprecate an API?**
- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by AnkurArohi** [comment]
Thanks a ton, looks nice!

File: index.html [hunk 1]
```diff
@@ -107,6 +107,19 @@ <h4>Dynawo</h4>
...
                 </div>
+                <div class="col-md-6 col-lg-4 portfolio-item filter-app">
+                    <div class="portfolio-wrap">
+                        <img src="/assets/img/portfolio/emsys_grid_logo_bigger.png" class="img-fluid" alt="emsys">
+                        <div class="portfolio-info">
+                            <h4>emsys grid</h4>
+                            <p>Modeling and exporting forecasted network state in CGMES format. CGMES export validation according to ENTSOE norms & operator inputs</p>
+                            <div class="portfolio-links">
+                                <a href="https://www.emsysgrid.com" title="More Details" target="_blank"><i class="bx bx-link"></i></a>
+                                <a href="" title="GitHub" target="_blank"><i class="bx bxl-github"></i></a>
```
Comment (colinepiloquet) [inline index.html:118]: This is the little GitHub icon right? Is it necessary to keep it if it does not lead anywhere? I would just put the link to the website maybe.

Comment (So-Fras) [reply index.html:118]: Yes, the github link is not available yet. I can put the website link for the moment!

Comment (AnkurArohi) [reply index.html]: thanks that will do , the github link is in process

**Review by colinepiloquet — APPROVED** [review]

**Pull request merged**